loadtest/user/userentity/user.go

```diff
@@ -33,12 +33,19 @@ type UserEntity struct {
 	wsServerSeq int64
 }
 
+const (
+	AuthenticationTypeMattermost = "mattermost"
```

Comment:
Should this be `AuthenticationTypeEmailPass`? 0/5

Reply:
+1